src/main.ts

```diff
@@ -13,6 +13,7 @@ export async function run(): Promise<void> {
     const secrets: string[] = core.getMultilineInput("secrets", {
       required: true,
     });
+    const cloudRegion = core.getInput("cloud_region");
     const baseUrl: string = core.getInput("base_url");
```

Comment:
Currently I'm investigating the scenario where cloudRegion is set AND baseUrl is set, in that scenario cloudRegion has no effect on the URL I'm not sure if this is obvious to the user that this would happen.